include/orb_slam_2_ros/interface_stereo.hpp

```diff
@@ -36,6 +41,13 @@ class OrbSlam2InterfaceStereo : public OrbSlam2Interface {
   std::shared_ptr<message_filters::Subscriber<sensor_msgs::Image>> left_sub_;
   std::shared_ptr<message_filters::Subscriber<sensor_msgs::Image>> right_sub_;
   std::shared_ptr<message_filters::Synchronizer<sync_pol>> sync_;
+
+  // Rectification maps
+  cv::Mat M1l_, M2l_, M1r_, M2r_;
```

Comment:
Thanks for the comment :+1:. But on second thought, can you rename these descriptively:

https://google.github.io/styleguide/cppguide.html#General_Naming_Rules